webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const HtmlPlugin = require('html-webpack-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
```

Comment:
```suggestion
  entry: ['@babel/polyfill', './src/index.js'],
```
Note in the [react-state.md](https://github.com/alchemy-fullstack-js-winter-2019/classwork/blob/master/4_react/3_reactState.md), that you need to install @babel/prefill to use the async/await syntax in App.js. Make sure you work on things in smaller chunks and try not to copy out code if you are unsure how it works.